addons/im_livechat/static/src/embed/common/chatbot/chatbot_model.js

```diff
@@ -71,7 +71,10 @@ export class Chatbot extends Record {
                 },
                 { html: true }
             );
-        this.thread.messages.add(this.currentStep.message);
+        }
+        if (this.currentStep.message?.notIn(this.thread.messages)) {
+            this.thread.messages.add(this.currentStep.message);
+        }
```

Comment:
`.add()` does nothing if the message is already in record list, so the `if ` is unnecessary